README.md

```diff
@@ -37,10 +44,10 @@
 | [000033](results/000033/status.yaml) | — | — | — |
 | [000034](results/000034/status.yaml) | [6 passed](results/000034/status.yaml#L9), 0 failed, 0 timed out | [1 passed](results/000034/status.yaml#L20), [1 failed](results/000034/status.yaml#L18), [4 timed out](results/000034/status.yaml#L22) | — |
 | [000035](results/000035/status.yaml) | [184 passed](results/000035/status.yaml#L9), 0 failed, [1 timed out](results/000035/status.yaml#L686) | [2 passed](results/000035/status.yaml#L777), [59 failed](results/000035/status.yaml#L693), [124 timed out](results/000035/status.yaml#L788) | — |
-| [000036](results/000036/status.yaml) | [57 passed](results/000036/status.yaml#L9), 0 failed, 0 timed out | 0 passed, [28 failed](results/000036/status.yaml#L261), [29 timed out](results/000036/status.yaml#L391) | — |
-| [000037](results/000037/status.yaml) | [47 passed](results/000037/status.yaml#L9), 0 failed, 0 timed out | 0 passed, [40 failed](results/000037/status.yaml#L243), [7 timed out](results/000037/status.yaml#L441) | — |
+| [000036](results/000036/status.yaml) | [57 passed](results/000036/status.yaml#L9), 0 failed, 0 timed out | 0 passed, [29 failed](results/000036/status.yaml#L261), [28 timed out](results/000036/status.yaml#L396) | — |
+| [000037](results/000037/status.yaml) | [47 passed](results/000037/status.yaml#L9), 0 failed, 0 timed out | 0 passed, [41 failed](results/000037/status.yaml#L243), [6 timed out](results/000037/status.yaml#L446) | — |
```

Comment:
results files didn't change but stats changed here, how come? (also in other spots)

Reply:
The README wasn't updated when a bunch of `results/` files were last updated in 2c6242c7.

Reply:
ah cool. Just wanted to make sure that code is not having some "flaky" point